syft/pkg/cataloger/apkdb/package.go

```diff
@@ -42,10 +42,8 @@ func packageURL(m pkg.ApkMetadata, distro *linux.Release) string {
 	}
 
 	return packageurl.NewPackageURL(
-		// note: this is currently a candidate and not technically within spec
-		// see https://github.com/package-url/purl-spec#other-candidate-types-to-define
+		packageurl.TypeAlpine,
 		"alpine",
-		"",
```

Comment:
Something about this seems off -- this is resulting in PURLs of the form: `apk/alpine/<package>`, should they be of this form or `apk/<package>`?

Reply:
nope, purl-spec says `pkg:apk/alpine/<package>`, same as for other types like `pkg:deb/ubuntu/<package>`

https://github.com/package-url/purl-spec/blob/master/PURL-TYPES.rst#apk

Reply:
Oh, excellent 👍